### ISSUE TITLE
UI: (13.C) Inventory Filters Applied has Insufficient Contrast (CRASM-375)

### ISSUE THREAD

**Issue by hawkishpolicy** [cisagov/XFD#378]
# 🐛 Summary #
Insufficient contrast is provided for “Filters Applied” text Inventory> Search Results page. The color difference of text and its background should have a contrast ratio of at least 4.5:1. Using contrast analyzers, spot test various text and backgrounds throughout the site or application to ensure a contrast ratio of at least 4.5:1. When insufficient contrast issues are found, adjust either one or the other color to meet a minimum of 4.5:1 ratio.

## Any helpful log output or screenshots ##

![image](https://github.com/cisagov/XFD/assets/123108455/597a1d75-9589-4a2b-a245-2b2f98283d68)
